src/assessments/links/test-steps/label-in-name-get-completed-requirement-details.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { LinksTestStep } from 'assessments/links/test-steps/test-steps';
+import {
+    AssessmentData,
+    GeneratedAssessmentInstance,
+    TestStepResult,
+} from 'common/types/store-data/assessment-result-data';
+import { ManualTestStatus } from 'common/types/store-data/manual-test-status';
+import { forEach } from 'lodash';
+
+const labelContainsVisibleText = (instance: GeneratedAssessmentInstance): boolean | undefined => {
+    const labelContainsVisibleText = instance.propertyBag['labelContainsVisibleText'];
+    if (labelContainsVisibleText === undefined) {
+        return undefined;
+    }
+    return labelContainsVisibleText === true;
+};
+
+export const labelInNameGetCompletedRequirementDetails = (assessmentData: AssessmentData) => {
+    let expectedPasses = 0;
+    let expectedFailures = 0;
+    let confirmedPasses = 0;
+    let confirmedFailures = 0;
+    forEach(Object.keys(assessmentData.generatedAssessmentInstancesMap), key => {
+        const instance: GeneratedAssessmentInstance =
+            assessmentData.generatedAssessmentInstancesMap[key];
+        const testStepResult: TestStepResult = instance.testStepResults[LinksTestStep.labelInName];
+        if (!testStepResult) {
+            return;
+        }
+        const status = testStepResult.status;
+        const expectedPass = labelContainsVisibleText(instance);
+
+        if (expectedPass === undefined) {
+            return;
+        }
+
+        if (expectedPass) {
```

Comment:
nit: I'm not sure if the expectedPass variable is necessary. I think it might actually read better using the function call inline ... so I read it like "if label contains visible text, increase the expected pass counter"

Reply:
Inlined